lib/importproject.cpp

```diff
@@ -167,18 +167,20 @@ void ImportProject::FileSettings::setIncludePaths(const std::string &basepath, c
 void ImportProject::import(const std::string &filename)
 {
     std::ifstream fin(filename);
+
+	std::string path(Path::getPathFromFilename(Path::fromNativeSeparators(filename)));
+	if (!path.empty() && !endsWith(path, '/'))
+		path += '/';
+
     if (!fin.is_open())
         return;
     if (filename.find("compile_commands.json") != std::string::npos) {
         importCompileCommands(fin);
     } else if (filename.find(".sln") != std::string::npos) {
-        std::string path(Path::getPathFromFilename(Path::fromNativeSeparators(filename)));
-        if (!path.empty() && !endsWith(path,'/'))
-            path += '/';
-        importSln(fin,path);
+        importSln(fin, path);
     } else if (filename.find(".vcxproj") != std::string::npos) {
         std::map<std::string, std::string> variables;
-        importVcxproj(filename, variables, emptyString);
+        importVcxproj(filename, path, variables, emptyString);
```

Comment:
this function call seems strange to me. If the filename is "a/b/c.vcxproj" then first parameter will be "a/b/c.vcxproj" and second parameter will be "a/b/". If "a/b/c.vcxproj" cant be loaded then as far as I see, importVcxproj() will try to load "a/b/a/b/c.vcxproj".

Am I missing something?